packages/react-on-rails-pro/src/index.ts

```
@@ -0,0 +1,164 @@
+  },
+
+  reactOnRailsStoreLoaded(storeName: string): Promise<void> {
+    return hydrateStore(storeName);
+  },
+};
+
+// Replace global ReactOnRails with Pro version
+globalThis.ReactOnRails = ReactOnRailsPro;
+
+// Pro client startup with immediate hydration support
+import { onPageLoaded, onPageUnloaded } from 'react-on-rails/pageLifecycle';
```

ESLint

[error] `react-on-rails/pageLifecycle` import should occur before import of `./ComponentRegistry.ts`

(import/order)